### PR TITLE
docs: add SECURITY.md

### PR DESCRIPTION
**Pull request by italvi**

### CODE REVIEW

**Review by mmarseu — DISMISSED** [review]
That's a great idea. I like it 👍 

Only one comment. I feel it would show a little more good faith on our part if we gave a deadline for public disclosure. For example, Google asks reporters only to wait up to 90 days before publicly disclosing their finding. Given the sort of problems we're likely going to face in this project, I believe that would be ample time for us to react.
Then again, if this document simply reflects the company-wide policy without any exceptions, then consider this point moot.

In any case, I approve.

**Comment by italvi** [comment]
@mmarseu I added "We aim to process your report within one week, and respond within two weeks.", so that the person reporting know how long them should wait till an answer. Would this also be fitting from your side? Google also [sometimes](https://github.com/google/osv.dev/security) only provide this information.

**Review by mmarseu — DISMISSED** [review]
LGTM

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+To report a security issue, please visit our website at [https://festo.com/psirt](https://festo.com/psirt) and follow the instructions provided. Our Product Security Incident Response Team (PSIRT) will review your report and respond accordingly.
+
+## Responsible Disclosure
+
+We kindly request that you adhere to responsible disclosure practices when reporting security issues to us. This includes:
+
+1. **Providing Sufficient Information**: Please provide detailed information about the vulnerability or concern you have identified. This will help our PSIRT team understand and address the issue more effectively.
+
+2. **Giving Us Time to Respond**: We appreciate your patience while our PSIRT team investigates and resolves the reported issue. We will make every effort to keep you informed of our progress. We aim to process your report within one week, and give you a first respond within two weeks. For critical vulnerabilities we aim to resolve them within 90 days.
```
Comment (CBeck-96) [inline SECURITY.md:17]: response?

**Review by CBeck-96 — APPROVED** [review]

**Review by mmarseu — APPROVED** [review]

**Pull request merged**